http/index.ts

```diff
@@ -16,6 +16,8 @@ import { HubSpotHttpError } from '../models/HubSpotHttpError';
 
 const i18nKey = 'http.index';
 
+export type HubSpotPromise<T = unknown> = AxiosPromise<T>;
```

Comment:
LDL currently defines all types within the `types` dir, so this should live in `types/http`